platform/darwin/src/MGLRasterSource.h

```diff
@@ -1,18 +1,57 @@
 #import "MGLSource.h"
 #import "MGLTypes.h"
 
+#import <CoreGraphics/CoreGraphics.h>
+
 @class MGLTileSet;
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ A raster tile source.
+ 
+ @see <a href="https://www.mapbox.com/mapbox-gl-style-spec/#sources-raster">The
+ style specification.</a>
+ */
 @interface MGLRasterSource : MGLSource
 
+/**
+ A URL to a TileJSON resource. Supported protocols are `http:`, `https:`, and 
+ `mapbox://<mapid>`.
+ */
 @property (nonatomic, readonly, copy) NSURL *URL;
+
+/**
+ The minimum visual size to display tiles for this source. 
+ */
 @property (nonatomic, readonly, assign) NSUInteger tileSize;
+
+/**
+ A tile set holds the raster tile URL template strings and associated
```

Comment:
Start off with "The tile set that the source uses to locate and present tiles." Or something like that. Just to make the relationship to the class clear.